style2.css

```diff
@@ -0,0 +1,189 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url("https://fonts.cdnfonts.com/css/Lato");
+
+* {
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.about-logo {
+  text-align: center;
+}
+
+.about-logo h2 {
+  font-family: 'Lato', sans-serif;
+  font-size: calc(1.3rem + 0.6vw);
+  font-weight: 500;
+  margin-top: 4rem;
+}
+
+.header-about {
+  font-family: 'cocogoose', sans-serif;
+  color: #ec5242;
+  font-size: 3rem;
+  padding-top: 5rem;
+  width: 40%;
+  margin: 0 auto 3rem auto;
+}
+
+.about-text {
+  width: 50rem;
+  background-color: #fff;
+  padding: 3rem;
+  justify-items: center;
+  margin: 30px 0;
+  margin: auto;
+  margin-bottom: 80px;
+  font-family: 'Lato', sans-serif;
+}
+
+.main-about {
+  background-image: url(./images/backbb.jpg);
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+  padding: 50px;
+}
+
+.footer2 {
+  background-color: #272a31 !important;
+  color: #fff;
+}
+
+.footer-container {
+  background-color: #272a31 !important;
+  color: #fff;
+}
+
+.about-2 {
+  background-color: #fff;
+  border-bottom: 1px solid gray;
+}
+
+.text p {
+  margin: auto;
+  padding: 0 20px;
+  font-family: 'Lato', sans-serif;
+  width: 50%;
+}
+
+.bar {
+  margin-bottom: 3rem;
+}
+
+.lt1 img {
+  width: 25%;
+  height: 25%;
+  margin: 4rem 0;
+}
+
+.lt2 {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+  margin: 4rem 0;
+}
+
+.lt23 img {
+  border: 2px solid #272a31;
+  width: 25rem;
+  height: 15rem;
+}
+
+.container-image {
+  position: relative;
+  text-align: center;
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+}
+
+.centered-text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background: rgb(199, 28, 28);
+  background: rgba(255, 18, 18, 0.5);
+  width: 25rem;
+  height: 100%;
+  padding-top: 60px;
+}
+
+.container-child-mobile {
+  display: none;
+}
+
+/* media query for about section */
+@media all and (max-width: 768px) {
+  .main-about {
+    padding: 27px 50px;
+  }
+
+  .about-text {
+    width: auto;
+    background-color: #fff;
+    padding: 1rem;
+    margin-bottom: 80px;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .header-about {
+    font-family: cocogoose, sans-serif;
+    color: rgb(236, 82, 66);
+    font-size: 2rem;
+    width: 86%;
+    margin: 0 auto 1rem;
+  }
+
+  .about-logo h2 {
+    font-size: 16px;
+  }
+
+  .lt2 {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    margin-bottom: 3rem;
+  }
+
+  .lt2 img {
+    width: 90%;
+  }
+
+  .centered-text {
+    width: 90%;
+  }
+
+  .text p {
+    width: auto;
+  }
+
+  .footer-wrap {
+    padding-top: 0;
+  }
+
+  .footer-container {
+    gap: 10px;
+  }
+
+  /* .footer-container {
+    display: none;
+  } */
```

Comment:
- [ ] Please kindly remove all commented code from your code base to make your code clean and easy to maintain.